src/main/java/com/dataloom/edm/internal/PropertyType.java

```diff
@@ -27,51 +28,71 @@ public PropertyType(
             @JsonProperty( SerializationConstants.TITLE_FIELD ) String title,
             @JsonProperty( SerializationConstants.DESCRIPTION_FIELD ) Optional<String> description,
             @JsonProperty( SerializationConstants.SCHEMAS ) Set<FullQualifiedName> schemas,
-            @JsonProperty( SerializationConstants.DATATYPE_FIELD ) EdmPrimitiveTypeKind datatype ) {
+            @JsonProperty( SerializationConstants.DATATYPE_FIELD ) EdmPrimitiveTypeKind datatype,
+            @JsonProperty( SerializationConstants.PII_FIELD ) boolean piiField ) {
```

Comment:
On the backend side there's currently a default value FALSE, so that all the old test codes will continue to work.

On the frontend side this would be tricky, and I am inclined to say NO. The main reason is that  officially there can only be one constructor marked with `JsonCreator`. If we need to give users the option of specifying the PII field, we must include it in the (only) constructor marked for JsonCreator.

If it's too much work to modify the front end for now, I can use a hacky way to expose two constructors marked for JsonCreator, but it is not officially documented and is unlikely to be a long-term fix.